applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -621,364 +866,157 @@ namespace Kratos
-		{
-			this->CalculateLumpedMassMatrix(rMassMatrix, rCurrentProcessInfo);
-		}
-		else
-		{
-			this->CalculateConsistentMassMatrix(rMassMatrix, rCurrentProcessInfo);
-			bounded_matrix<double,msElementSize,msElementSize> rotation_matrix = ZeroMatrix(msElementSize,msElementSize);
-			bounded_matrix<double,msElementSize,msElementSize> aux_matrix = ZeroMatrix(msElementSize,msElementSize);
-
-			rotation_matrix = this->mRotationMatrix;
-			aux_matrix = prod(rotation_matrix, rMassMatrix);
-			rMassMatrix = prod(aux_matrix,
-				Matrix(trans(rotation_matrix)));
-		}
-		KRATOS_CATCH("")
-	}
-
-	void CrBeamElement3D2N::CalculateDampingMatrix(MatrixType& rDampingMatrix,
-		ProcessInfo& rCurrentProcessInfo) {
-
-		KRATOS_TRY
-		if (rDampingMatrix.size1() != msElementSize)
-		{
-			rDampingMatrix.resize(msElementSize, msElementSize, false);
+		Quaternion<double> q(r0, r1, r2, r3);
+		q.RotateVector3(rotated_nx0);
+		q.RotateVector3(rotated_ny0);
+		q.RotateVector3(rotated_nz0);
+
+
+		bounded_matrix<double,msDimension,msDimension> rotated_coordinate_system = ZeroMatrix(msDimension, msDimension);
+		for (unsigned int i = 0; i < msDimension; ++i) {
+			rotated_coordinate_system(i, 0) = rotated_nx0[i];
+			rotated_coordinate_system(i, 1) = rotated_ny0[i];
+			rotated_coordinate_system(i, 2) = rotated_nz0[i];
 		}
 
-		rDampingMatrix = ZeroMatrix(msElementSize, msElementSize);
+		//rotate basis to element axis + redefine R
+		Bisectrix = ZeroVector(msDimension);
+		Vector delta_x = ZeroVector(msDimension);
+		double vector_norm;
 
-		Matrix stiffness_matrix = ZeroMatrix(msElementSize, msElementSize);
 
-		this->CalculateLeftHandSide(stiffness_matrix, rCurrentProcessInfo);
+		bounded_vector<double,msLocalSize> current_nodal_position = this->GetCurrentNodalPosition();
+		for (unsigned int i = 0; i<msDimension; ++i) delta_x[i] = current_nodal_position[msDimension+i] - current_nodal_position[i];
 
-		Matrix mass_matrix = ZeroMatrix(msElementSize, msElementSize);
 
-		this->CalculateMassMatrix(mass_matrix, rCurrentProcessInfo);
+		vector_norm = MathUtils<double>::Norm(delta_x);
+		if (vector_norm > numerical_limit) delta_x /= vector_norm;
 
-		double alpha = 0.0;
-		if (this->GetProperties().Has(RAYLEIGH_ALPHA))
-		{
-			alpha = this->GetProperties()[RAYLEIGH_ALPHA];
-		}
-		else if (rCurrentProcessInfo.Has(RAYLEIGH_ALPHA))
-		{
-			alpha = rCurrentProcessInfo[RAYLEIGH_ALPHA];
-		}
 
-		double beta = 0.0;
-		if (this->GetProperties().Has(RAYLEIGH_BETA))
-		{
-			beta = this->GetProperties()[RAYLEIGH_BETA];
-		}
-		else if (rCurrentProcessInfo.Has(RAYLEIGH_BETA))
-		{
-			beta = rCurrentProcessInfo[RAYLEIGH_BETA];
+		Bisectrix = rotated_nx0 + delta_x;
+		vector_norm = MathUtils<double>::Norm(Bisectrix);
+		if (vector_norm > numerical_limit) Bisectrix /= vector_norm;
+
+		bounded_matrix<double,msDimension,msDimension> n_xyz = ZeroMatrix(msDimension);
+		for (unsigned int i = 0; i < msDimension; ++i) {
+			n_xyz(i, 0) = -1.0 * rotated_coordinate_system(i, 0);

```

Comment:
take out the multiplication by 1.0. use instead

        n_xyz(i, 0) = - rotated_coordinate_system(i, 0);
	n_xyz(i, 1) = rotated_coordinate_system(i, 1);
        ....